winui/BusyIndicator/Getting-Started.md

```diff
@@ -1,12 +1,114 @@
 ---
 layout: post
-title: Getting Started with WinUI BusyIndicator control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control, its elements, and more here.
+title: Getting started with WinUI BusyIndicator control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control and its basic features.
 platform: WinUI
 control: SfBusyIndicator
 documentation: ug
 ---
 
 # Getting Started with WinUI BusyIndicator
 
-This section explains the steps required to add the BusyIndicator control and covers only the basic features needed to get started with Syncfusion SfBusyIndicator control.
\ No newline at end of file
+This section explains the steps required to add the BusyIndicator control and covers only the basic features needed to get started with Syncfusion `BusyIndicator` control.
+
+## Creating an application with WinUI BusyIndicator control
+
+1. Create a [WinUI 3 desktop app for C# and .NET 5](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to [Syncfusion.Notifications.WinUI](https://www.nuget.org/packages/Syncfusion.Notifications.WinUI) NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Notifications` in XAML or C# code.
+4. Initialize the SfBusyIndicator control.
+
+{% tabs %}
+{% highlight xaml %}
+
+<Page
+    x:Class="GettingStarted.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:notification="using:Syncfusion.UI.Xaml.Notifications"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+      <notification:SfBusyIndicator IsActive="True"/>
+    </Grid>
+</Page>
+
+{% endhighlight %}
+{% highlight C# %}
+
+// Creating an instance of the BusyIndicator control.
+SfBusyIndicator busyIndicator = new SfBusyIndicator();
+
+// Activating the SfBusyIndicator.
+busyIndicator.IsActive = true;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Animation Type in BusyIndicator
+
+The BusyIndicator control provides 7 predefined animation types like `DottedCircularFluent`, `DottedCircle`, `DottedLinear` and so on. User can select any one of the animation types using `AnimationType` property.
```

Comment:
The BusyIndicator control provides 7 predefined animation types like `DottedCircularFluent`, `DottedCircle`, `DottedLinear`, and so on. Users can select any one of the animation types using the `AnimationType` property.

Reply:
Changed the animation type content as mentioned above.